include/boost/spirit/include/qi_expectd.hpp

```diff
@@ -0,0 +1,19 @@
+/*=============================================================================
+    Copyright (c) 2001-2011 Joel de Guzman
+    Copyright (c) 2001-2011 Hartmut Kaiser
+    Copyright (c) 2016      Frank Hein, maxence business consulting gmbh
+    http://spirit.sourceforge.net/
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+=============================================================================*/
+#ifndef BOOST_SPIRIT_INCLUDE_QI_EXPECTD
+#define BOOST_SPIRIT_INCLUDE_QI_EXPECTD
```

Comment:
I still do not understand what you mean. What does X3 differently? What am I missing?